index.html

```diff
@@ -143,46 +142,46 @@ <h1 id="title">[PROPOSED] Verifiable Credentials Working Group Charter</h1>
         </table>
       </section>
 
-      <!-- MARK: Scope -->
-      <section id="scope" class="scope">
-        <h2>Scope</h2>
+      <!-- MARK: Background -->
+      <section id="background" class="background">
+        <h2>Motivation and Background</h2>
         <p>
-          Building on the experience gained through implementation, deployment and usage
-          of Verifiable Credentials (VCs), this Working Group will extend Verifiable
-          Credential foundations with new standardized technologies to improve the use of
-          this technology on the Web.
+          In the past few years the Working Group has developed a family of specifications in the area of Verifiable Credentials. 
+          They provide a mechanism to express "credentials" 
+          (driver's licenses, university degrees, government-issued residence permits, etc.) in a digital 
+          format on the Web that is cryptographically secure, privacy respecting, and machine-verifiable.
+          These technologies are being gradually adopted by various institutions, governmental or otherwise, in different 
+          countries around the globe.
         </p>
+        
         <p>
-          The <b>scope</b> of the Verifiable Credentials Working Group is:
+          Readers that are new to this work should read the <a href="https://www.w3.org/TR/vc-overview/">Verifiable Credentials Overview</a> W3C Note, published by the Working Group. The Note help to understand better some the terminology used in this charter e.g., Verifiable Credentials and Presentation, Data Integrity, or cryptosuites. It also provides a technical overview on how the various specifications listed as
```

Comment:
```suggestion
          Readers that are new to this work should read the <a href="https://www.w3.org/TR/vc-overview/">Verifiable Credentials Overview</a> W3C Note, published by the Working Group. The Note explains some the terminology used in this charter e.g., Verifiable Credentials and Presentation, Data Integrity, or cryptosuites. It also provides a technical overview on how the various specifications listed as
```

Reply:
This includes @brentzundel's suggestion.
```suggestion
          Readers that are new to this work should read the <a href="https://www.w3.org/TR/vc-overview/">Verifiable Credentials Overview</a> W3C Note, published by the Working Group. The Note explains some of the terminology used in this charter, e.g., Verifiable Credentials and Presentations, Data Integrity, and cryptosuites. It also provides a technical overview of how the various specifications listed as
```

Reply:
Actually, that whole section has been removed, except that, until now, I kept in the source in comments. I did that to make life easier if someone wants to keep it. But I will remove it.